nevergrad/optimization/recastlib.py

```diff
@@ -57,18 +57,36 @@ def _optimization_function(self, objective_function: tp.Callable[[tp.ArrayLike],
         if self.initial_guess is not None:
             best_x = np.array(self.initial_guess, copy=True)  # copy, just to make sure it is not modified
         remaining: float = budget - self._num_ask
+
+        def smac_obj(p):
+            data = [np.arctanh(p["x" + str(i)]) for i in range(self.dimension)]
```

Comment:
just like in the other PR, arctan seems a better fit

Reply:
done